### PR TITLE
fix(client): update layout once per properties panel resize

### LINKED ISSUE #1369: Poor Performance When Resizing Properties Panel
When resizing the properties panel [we're updating the React application's state](https://github.com/camunda/camunda-modeler/blob/master/client/src/app/tabs/PropertiesContainer.js#L98) quite a lot. [Throttling](https://github.com/camunda/camunda-modeler/blob/master/client/src/app/tabs/PropertiesContainer.js#L39) is helping a little bit but the issue still exists. We should avoid state updates until resizing is finished.

---

Related to [SUPPORT-5769](https://app.camunda.com/jira/browse/SUPPORT-5769)

### PR DESCRIPTION
**Pull request by philippfromme**
Closes #1369

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: client/src/app/tabs/PropertiesContainer.js [hunk 1]
```diff
@@ -12,88 +12,83 @@ import React, { PureComponent } from 'react';
...
 };
 
+export const MIN_WIDTH = 150;
+export const MAX_WIDTH = 650;
 
-/**
- * A generic container to hold our editors properties panels.
- *
- * Adds resize and toggle support.
- */
```
Comment (barmac) [inline client/src/app/tabs/PropertiesContainer.js:33]: Why's documentation gone? :(

**Comment by barmac** [comment]
resize

Let's make sure we have a few test cases for series of resize. E.g.

```
resize to max -> start resize
resize to zero -> start resize
```

**Comment by philippfromme** [comment]
The issue was that I forgot to add the `onDragEnd` listener. Since we test the API excluding the DOM interaction this didn't break the tests.

File: client/src/app/tabs/PropertiesContainer.js (hunk 1 above)
Comment (philippfromme) [reply client/src/app/tabs/PropertiesContainer.js]: Sorry, back again.

**Comment by philippfromme** [comment]
@barmac Let me know if this is okay for you to merge.

**Pull request merged**